css/properties.json

```diff
@@ -4772,6 +4743,52 @@
     "order": "uniqueOrder",
     "status": "standard"
   },
+  "offset-distance": {
+    "syntax": "&lt;length-percentage&gt;",
+    "media": "visual",
+    "inherited": false,
+    "animationType": "lpc",
+    "percentages": "referToTotalPathLength",
+    "groups": [
+      "CSS Motion"
+    ],
+    "initial": "<code>0</code>",
+    "appliesto": "allElementsSVGContainerElements",
+    "computed": "asSpecified",
+    "order": "uniqueOrder",
+    "status": "experimental"
+  },
+  "offset-path": {
+    "syntax": "&lt;url&gt; | [ &lt;basic-shape&gt; | &lt;path()&gt; ] || &lt;geometry-box&gt; | none",
+    "media": "visual",
+    "inherited": false,
+    "animationType": "discrete",
```

Comment:
The spec. now says "as <angle>, <basic-shape> or <path()>". So, this needs to be replaced by a new keyword and you need to add a new entry to the [l10n/css.json](https://github.com/mdn/data/blob/master/l10n/css.json).

Sebastian